google_drive/public/js/google_calendar.js

```diff
@@ -1,12 +1,15 @@
 /* Javascript for GoogleDocumentBlock. */
 function GoogleCalendarBlock(runtime, element) {
 
+    var ajaxflag = $('.ajaxflag', element);
```

Comment:
I have no idea what "```ajaxflag```" is! It appears to exist solely for the purpose of altering content upon completion of publishing the event; I assume this is for testing purposes, so that one can say something along the lines of:
"I've published the event that indicates that the load has completed".

There's a couple of things of note herein:
i) The name is meaningless - I'd choose something more meaningful, such as ```load_event_complete```
ii) The variable is only used once, seems to me that it would be much easier to read the code if the selector was simply used in the one place in which it is used.